tools/update-prod-static

```diff
@@ -87,12 +87,10 @@ subprocess.check_call(['./manage.py', 'compilemessages'],
 
 # Move the source maps out of the serve/ directory and into their
 # proper place.
-subprocess.check_call(['rm', '-rf', 'prod-static/source-map'],
-                      stdout=fp, stderr=fp)
 subprocess.check_call(['mkdir', '-p', 'prod-static'],  # Needed if PRODUCTION
                       stdout=fp, stderr=fp)
-subprocess.check_call(['mv', os.path.join(settings.STATIC_ROOT, 'source-map'),
-                       'prod-static/source-map'],
+subprocess.check_call(['cp', '-r', os.path.join(settings.STATIC_ROOT, 'source-map'),
+                       'prod-static'],
```

Comment:
Should this be `cp -rT prod-static/source-map` here for better readability?  I guess it's guaranteed to be copying in because of the `mkdir` the line before.... 

Pinging @gnprice for thoughts on this since he found a similar issue that was a bug recently (I don't remember the details).